source/features/release-download-count.tsx

```diff
@@ -33,7 +33,7 @@ async function addCounts(assetsList: HTMLElement): Promise<void> {
 		?? assetsList.closest('.Box:not(.Box--condensed)')!; // Releases list, excludes the assets list’s own .Box
 
 	const releaseName = container
-		.querySelector('.octicon-tag ~ span')!
+		.querySelector('.Link[href*="/tree/"]')!
```

Comment:
Does this work on the /fregante/tree repo?

I'd probably rather just add both icon here with a mention of the other feature.

.querySelector accepts an array, which makes it easy to comment